js/book-carousel.js

```diff
@@ -1,6 +1,7 @@
 namespacer('bcpl');
 
-bcpl.bookCarousel = (($, constants) => {
+bcpl.bookCarousel = (($, constants, imageAutoWidth) => {
```

Comment:
imageAutoWidth is imported but never used.

Reply:
done